client_plugin/drivers/shared/kvstore/kvstore.go

```diff
@@ -0,0 +1,75 @@
+// Copyright 2017 VMware, Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package kvstore
+
+// VolStatus: Datatype for keeping status of a shared volume
+type VolStatus string
+
+/*
```

Comment:
Have we documented/added comments for the possible state transition? It would be very helpful for others to understand the code